lib/export-scala-regmap.js

```diff
@@ -120,8 +120,11 @@ const generateRegister = (access, register, addressBlockNames) => {
 
   const registerDesc = register.description ? `Some("${register.description}")` : 'None';
 
+  const addressOffset =
+    addressUnitBits == 8 ? register.addressOffset : `${register.addressOffset} * ${register.addressSizeUnit}`;
```

Comment:
```suggestion
    addressUnitBits === 8 ? register.addressOffset : `${register.addressOffset} * ${register.addressSizeUnit}`;
```

Remember to use `===` in JavaScript for equality comparisons, because otherwise you end up with automatic type coercions like `"8" == 8` returning `true`.